matlab/examples/AeroShield/AeroShield_PID/AeroShield_PID.m

```diff
@@ -0,0 +1,82 @@
+% clear all objects and variables
+clear all
+clc 
+
+% initialisation of AeroShield library 
+AeroShield=AeroShield; 
+
+% create an arduino object + hall sensor object 
+AeroShield.begin();
+
+% calibration
+startangle= AeroShield.calibration(); 
+lastangle=startangle+1024; % save startangle value + 1024(90°).(for mapping purpose)
+
+% initialisation of PID library 
+PID = PID;
+
+Ts = 0.002;             % sampling period           
+Kp=0.035 ;              % PID Gain
+Ti=0.00038 ;            % PID Integral time constant
```

Comment:
That is some very careful tuning:) Its okay, im just in awe, how exact you have gone. This raises a question: Imagine you create 100 Aero Shield devices. Would this tunign be good for all of them?:)

Reply:
oh hell no.... it wont work.... I was tunning that shit for 3 days and just now, retune it.... It works better now, with a bit more normal-ish numbers..